source/MRViewer/MRPalette.cpp

```diff
@@ -325,10 +332,15 @@ void Palette::draw( const ImVec2& pose, const ImVec2& size )
     }
     
     const auto& style = ImGui::GetStyle();
-    const auto& windowSize = Viewer::instance()->viewport().getViewportRect();
+    const auto& viewer = Viewer::instanceRef();
```

Comment:
better use new function
```suggestion
    const auto& viewer = getViewerInstance();